composer.json

```diff
@@ -22,17 +22,23 @@
         "issues": "http://github.com/liip/RMT/issues"
     },
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.6|^7.0",
         "symfony/console": "~2.3|^3.0",
         "symfony/yaml": "~2.3|^3.0",
         "symfony/process": "~2.3|^3.0",
         "vierbergenlars/php-semver": "~3.0",
         "sensiolabs/security-checker": "~2.0|^3.0|^4.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^5.6.8|^6.0"
```

Comment:
I thing 6.0 will break the tests cause of non BC compatible test classes and methods. Methods are now static called. Additional it should be ^5.6 to receive some patches for 5.6 branch.